package.json

```diff
@@ -124,7 +124,7 @@
     "@typescript-eslint/parser": "^5.59.0",
     "add": "^2.0.6",
     "autoprefixer": "^10.4.4",
-    "aws-sdk-client-mock": "^0.6.2",
+    "aws-sdk-client-mock": "^2.1.1",
```

Comment:
This one does not have Jest as a dependency so I figured we could keep it. It is the latest version available.